src/asvz_bot.py

```diff
@@ -145,8 +145,8 @@ def from_lesson_attributes(
                 weekday_date.year,
                 weekday_date.month,
                 weekday_date.day,
-                start_time.hour,
-                start_time.minute,
+                f"{start_time.hour:02d}",
```

Comment:
Would you mind to either put the formatting on L141 or move the formatting there down here? I would like to keep it consistent.